doc/src/site/sphinx/ml/sw_kmeans.rst

```diff
@@ -102,6 +102,7 @@ The following sections describe how to train the KMeans model in Sparkling Water
 
 H2O KMeans Parameters
 ~~~~~~~~~~~~~~~~~~~~~
+See also :ref:`parameters_H2OKmeans`.
 
 - **maxIterations**
```

Comment:
Should we keep the parameters listed bellow?

Reply:
I was also thinking about that but I'm not sure whether it's better remove it or not. The parameter descriptions comes from H2O-3 V3 classes and sometimes give very little pieces of information. On the other hand, the link contains a complete list of parameters.

I will do whatever you prefer...

Reply:
Let's keep it. Small duplication is better then missing documentation. WDYT?